subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -206,11 +252,48 @@ int lll_adv_data_release(struct lll_adv_pdu *pdu)
 	return 0;
 }
 
+static struct pdu_adv *adv_pdu_allocate(struct lll_adv_pdu *pdu, uint8_t last)
```

Comment:
Move all static functions in the logical order of execution after all the non-static functions in this file.

Reply:
Code moved according to suggestion.

What do you thing about extract of functions responsible for PDU memory handling into separate file e.g. lll_adv_pdu.c?
The code servers now legacy, extended and periodic advertising. It may improve readability of the lll_adv.c source code.

Reply:
No objections to  have pdu operations in separate file, but do it in a separate PR? (just being cautious about merge conflicts to other team members working on same file)

Reply:
Yes, separate PR is better option. Lets postpone it until 2.5 is released, then we can schedule it when all team members are ok to make this change.